openc3/lib/openc3/utilities/store_autoload.rb

```diff
@@ -202,7 +202,6 @@ def read_topics(topics, offsets = nil, timeout_ms = 1000, count = nil)
     # @return [String] the entry id
     def write_topic(topic, msg_hash, id = '*', maxlen = nil, approximate = 'true')
       id = '*' if id.nil?
-      Logger.debug "write_topic topic:#{topic} id:#{id} hash:#{msg_hash}"
```

Comment:
This results in a deadlock because Logger is calling write_topic within log_message